browser/models.py

```diff
@@ -506,3 +510,41 @@ def _history_user(self):
     @_history_user.setter
     def _history_user(self, value):
         self.changed_by = value
+
+
+class Position(YearMixin, CuratedMixin, LastUpdatedMixin):
+    history = HistoricalRecords()
+    person = models.ForeignKey('Person')
+    subject = models.CharField(max_length=255, blank=True)
+    CORP = 'Corporation Member'
+    TRUSTEE = 'Trustee'
+    FEL = 'Friday Evening Lecturer'
+    role_choices = ((CORP, 'Corporation Member'), (TRUSTEE, 'Trustee'),
```

Comment:
Please see the django documentation for an example of this. 'Corporation Member' is not a good 'key'. way to easy to be confused with the label.